content/2_Bronze/Intro_Sets.mdx

```diff
@@ -247,88 +247,6 @@ print(len(distinct_nums))
 
 </LanguageSection>
```

Comment:
Now there's a method 1 without a method 2. You should link back to the previous method.